bin/init-cluster

```diff
@@ -22,10 +52,17 @@ fi
 
 
 if ! k3d cluster list "${K3D_CLUSTER_NAME}" &> /dev/null ; then
+
+  # Ensure that the longhorn directory exists on the host
```

Comment:
longhorn?

Reply:
Oopsie, this is a residue of a test I made with a storageClass named Longhorn, that doesn't work with k3d ;)